_tests_/Ship.test.js

```diff
@@ -0,0 +1,66 @@
+const Port = require("../src/Port.js");
+const Ship = require("../src/Ship.js");
+const Itinerary = require("../src/Itinerary.js");
```

Comment:
Since `Ship` is the only component being used, the other imports can be deleted
```suggestion
const Ship = require("../src/Ship.js");
```